modules/apps/portal/portal-tika/src/main/java/com/liferay/portal/tika/internal/metadata/TikaRawMetadataProcessor.java

```diff
@@ -330,7 +330,7 @@ private Metadata _postProcessMetadata(String mimeType, Metadata metadata) {
 			_addFields(Office.class, fields);
 			_addFields(OfficeOpenXMLCore.class, fields);
 			_addFields(TIFF.class, fields);
-			_addFields(TikaMetadataKeys.class, fields);
+			_addFields(TikaCoreProperties.class, fields);
```

Comment:
also need a official doc/link in the commit message about TikaMetadataKeys changed to TikaCoreProperties